pkg/services/mssqlfg/failover_group_arm_template.go

```diff
@@ -0,0 +1,41 @@
+package mssqlfg
+
+// nolint: lll
+var failoverGroupARMTemplateBytes = []byte(`
+{
+	"$schema": "http://schema.management.azure.com/schemas/2014-04-01-preview/deploymentTemplate.json#",
+	"contentVersion": "1.0.0.0",
+	"parameters": {
+		"tags": {
+			"type": "object"
+		}
+	},
+	"resources": [
+		{
+			"type": "Microsoft.Sql/servers/failoverGroups",
+			"name": "{{ .priServerName}}/{{ .failoverGroupName }}",
+			"apiVersion": "2015-05-01-preview",
+			"properties": {
+	      "readWriteEndpoint": {
```

Comment:
I remember it was made an parameter in MASB, but seems not configurable here.

Reply:
Yes. That's a recent change in MASB. But it is for a temporary remedy. If Azure SQL doesn't have the issue, the customer wouldn't need this. In OSBA, let's aim at a mature situation as the issue will eventually fixed by SQL team. Also, we can still expose the parameter as an optional one when it is needed. For now, I think we can keep it as simple as possible.